src/main/java/rlbot/render/NamedRenderer.java

```diff
@@ -0,0 +1,22 @@
+package rlbot.render;
+
+import com.google.flatbuffers.FlatBufferBuilder;
+import rlbot.cppinterop.RLBotDll;
+
+public class NamedRenderer extends Renderer {
+    public NamedRenderer(final String renderName) {
+        super(renderName.hashCode());
+    }
+
+    public void startPacket() {
+        builder = new FlatBufferBuilder(1000);
+    }
+
+    public RenderPacket finishPacket() {
```

Comment:
is `finishPacket` going to be used without `finishAndSend()` ? If not, can it be private?

Reply:
Yeah, I wanted to allow the possibility of finishing without sending. People can reuse certain renderings that way.

Reply:
Nice. Thanks.